docs/guide_trivia_list_creation.rst

```diff
@@ -41,12 +41,32 @@ If there are multiple authors, we can separate them with commas.
 
     AUTHOR: Red, Rojo, Rouge
 
+-----------------
+Description Field
+-----------------
+
+We can also add an optional ``DESCRIPTION`` to our trivia list, which
+will show from the output of ``[p]trivia info <category>``. THe
+description should indicate to the user what the trivia list is
+about and what kind of questions they can expect to face.
+
+For example, if you were writing a logo quiz trivia list, you could
+create a description like this:
+
+.. code-block:: yaml
+
+    AUTHOR: Kreusada
+    DESCRIPTION: A quiz to test your logo knowledge to the limit. This trivia
+        will send image URLs, and you have to identify the company's name from
+        the logo that is sent.
```

Comment:
Multi-line plain scalars can be a foot gun, I think it's better to just show usage of block scalars:
```suggestion
    DESCRIPTION: >-
      A quiz to test your logo knowledge to the limit. This trivia
      will send image URLs and ask you to identify the company's name
      from the logo that is sent.
```
(also did a slight rewording because the tenses seemed inconsistent)